### PR TITLE
added docker config

### PR DESCRIPTION
**Pull request by JonghunYu**

### CODE REVIEW

**Comment by JonghunYu** [comment]
@vinayr If you have some time, please review this PR.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: '3'
```
Comment (vinayr) [inline docker-compose.yml]: If you plan to use ECS then be aware that `ecs-cli` compose do not support docker-compose file version 3. Also it does not support all the parameters, for e.g `depends_on` (`depends_on` is `links` in ecs) https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cmd-ecs-cli-compose.html

File: docker/app/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.4.2
```
Comment (vinayr) [inline docker/app/Dockerfile]: Try `alpine` images. They are small compared to default image. Same for `nginx`

File: docker-compose.yml (hunk 1 above)
Comment (JonghunYu) [reply docker-compose.yml]: oh Thanks my docker master haha

File: docker/app/Dockerfile (hunk 2 above)
Comment (JonghunYu) [reply docker/app/Dockerfile]: will take a look Thanks

**Pull request merged**